File: src/main/java/com/neutralplasma/virtusbot/storage/TicketInfo.java
package com.neutralplasma.virtusbot.storage;

public class TicketInfo {

    String userid = "";
    String channelID = "";

    public TicketInfo(String userid, String channelID){
        this.userid = userid;
        this.channelID = channelID;
    }


    public String getUserid(){
        return this.userid;
    }

    public String getChannelID(){
        return this.channelID;
    }

}

File: src/main/java/com/neutralplasma/virtusbot/storage/ticket/TicketInfo.kt
package com.neutralplasma.virtusbot.storage.ticket

class TicketInfo(userID: String, channelID: String, closed: Int, guildID: String) {
    var userID = ""
    var channelID = ""
    var guildID = ""
    var closed = 0

    init {
        this.userID = userID
        this.channelID = channelID
        this.closed = closed
        this.guildID = guildID
    }
}